### PR TITLE
add sql statement for show analyze status

### PR DESCRIPTION
**Pull request by wshwsh12**
### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [x] v3.1 (TiDB 3.1 versions)
- [x] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

File: sql-statements/sql-statement-show-analyze-status.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+---
+title: SHOW ANALYZE STATUS
+summary: TiDB 数据库中 SHOW ANALYZE STATUS 的使用概况。
+category: reference
+aliases: ['/docs-cn/dev/reference/sql/statements/show-analyze-status/']
```
Comment (bb7133) [inline sql-statements/sql-statement-show-analyze-status.md:5]: Please remove this line.

Comment (wshwsh12) [reply sql-statements/sql-statement-show-analyze-status.md]: Addressed.

File: sql-statements/sql-statement-show-analyze-status.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
+---
+title: SHOW ANALYZE STATUS
```
Comment (TomShawn) [inline sql-statements/sql-statement-show-analyze-status.md:2]: 请添加到 TOC.

File: sql-statements/sql-statement-show-analyze-status.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+summary: TiDB 数据库中 SHOW ANALYZE STATUS 的使用概况。
+category: reference
+---
+
+# SHOW ANALYZE STATUS
+
+`SHOW ANALYZE STATUS` 语句提供 TiDB 正在执行的收集统计信息的任务以及有限条历史任务记录。
```
Comment (TomShawn) [inline sql-statements/sql-statement-show-analyze-status.md:9]:
```suggestion
`SHOW ANALYZE STATUS` 语句提供 TiDB 正在执行的统计信息收集任务以及有限条历史任务记录。
```

File: sql-statements/sql-statement-show-analyze-status.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+| test         | t          | p3             | analyze index idx |              0 | 2020-05-25 17:23:55 | finished |
+| test         | t          | p3             | analyze columns   |              0 | 2020-05-25 17:23:55 | finished |
+| test         | t          | p2             | analyze columns   |              0 | 2020-05-25 17:23:55 | finished |
++--------------+------------+----------------+-------------------+----------------+---------------------+----------+
+8 rows in set (0.00 sec)
+```
+
+## 另请参阅
+
+* [ANALYZE_STATUS 表](/system-tables/system-table-information-schema.md#ANALYZE_STATUS-表)
```
Comment (TomShawn) [inline sql-statements/sql-statement-show-analyze-status.md:49]:
```suggestion
* [ANALYZE_STATUS 表](/system-tables/system-table-information-schema.md#analyze_status-表)
```

File: sql-statements/sql-statement-show-analyze-status.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+![ShowTargetFilterable](/media/sqlgram/ShowTargetFilterable.png)
+
+## 示例
+
+{{< copyable "sql" >}}
+
+```sql
+create table t(x int, index idx(x)) partition by hash(x) partition 4;
+analyze table t;
+show analyze status;
```
Comment (TomShawn) [inline sql-statements/sql-statement-show-analyze-status.md:28]: 几条语句不需要分开写？

Comment (wshwsh12) [reply sql-statements/sql-statement-show-analyze-status.md:28]: 我认为不需要分开写，这几条语句是可以让用户直接复制到客户端进行验证的

**Pull request merged**